Makefile

```diff
@@ -22,6 +22,17 @@
 help:     						## Show this help.
 	@sed -ne '/@sed/!s/## //p' $(MAKEFILE_LIST)
 
+pytest:
+	-rm -r .featureform
```

Comment:
just curious, what's the difference between `-rm` and `rm`?

Reply:
-rm allows it to fail, so if that directory doesn't exist, then it skips it